parser/src/defs.rs

```diff
@@ -3739,7 +3739,11 @@ sys_op! {
     /// ex: &cl &w "Hello, world!" . &fc "file.txt"
     ///   : &fras "file.txt"
     (2(0), Write, Stream, "&w", "write", Mutating),
-    /// Moves to a absolute position in the file under the stream.
+    /// Moves to an absolute position in a file stream
```

Comment:
Sorry to be this nitpicky, but other docs pages are generally imperative, not declarative.
```suggestion
    /// Move to an absolute position in a file stream
```